app/services/idv/steps/doc_auth_base_step.rb

```diff
@@ -58,9 +58,11 @@ def idv_failure(result)
       end
 
       def verify_back_image(reset_step:)
-        back_image_verified, data, analytics_hash = assure_id_results
-        data[:notice] = I18n.t('errors.doc_auth.general_info') if data.class == Hash
-        return friendly_failure(data, analytics_hash) unless back_image_verified
+        general_info_message = I18n.t('errors.doc_auth.general_info')
+
+        back_image_verified, data = assure_id_results
```

Comment:
would it be too invasive to turn this into a struct? that makes it much harder to mess up which value is which

ex

```
assure_id_results.back_image_verified
```

Reply:
I can come back around and try that. Right now this is a pattern that is pretty common. It would be tricky for me to go through and clean it up quickly.

Reply:
Totally more important to fix this now, but ya definitely something to consider as a follow-up